restclient/restclient.go

```diff
@@ -95,8 +97,10 @@ func (c *Client) Run(ctx context.Context, resp interface{}) (err error) {
 		return fmt.Errorf("reading body: %w", err)
 	}
 
-	if err := json.Unmarshal(body, &resp); err != nil {
-		return fmt.Errorf("unmarshall: %w", err)
+	if c.method != http.MethodDelete {
```

Comment:
Does this need to be run for all other verbs i.e PUT/PATCH or just for GET requests?

Reply:
Only using GET and DELETE thus far